### PR TITLE
add Quarkus Java Embedded Framework to Quarkus extension catalog

### PR DESCRIPTION
**Pull request by alexeysharandin**

### CODE REVIEW

**Review by gastaldi — CHANGES_REQUESTED** [review]
Let's wait until a release is out to merge this

**Comment by alexeysharandin** [comment]
@gastaldi , please merge. Repo ready

**Comment by gastaldi** [comment]
Thanks!

**Pull request merged**
